UlKit/UlKit/Feed/Cell/FeedRecommendationsViewCell.swift

```diff
@@ -0,0 +1,137 @@
+// FeedRecommendationsViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Настройка для ячейки рекомендаций
+final class FeedRecommendationsViewCell: UITableViewCell {
+    // MARK: - Constants
+
+    enum Constants {
+        static var spacingX: CGFloat = 10
+        static var fontVerdana = "Verdana"
+        static var fontVerdanaBold = "Verdana-Bold"
+    }
+
+    // MARK: - Visual Components
+
+    private let scrollView: UIScrollView = {
+        let scrollView = UIScrollView()
+        scrollView.showsHorizontalScrollIndicator = false
+        scrollView.isPagingEnabled = true
+        scrollView.translatesAutoresizingMaskIntoConstraints = false
+        return scrollView
+    }()
+
+    private let recommendLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Рекомендуем вам"
```

Comment:
```suggestion
        label.text = ""
```
по файлу ниже пробегись